utils.go

```diff
@@ -118,3 +120,15 @@ func ReadKernelCmdline() string {
 func IsKernelCmdlineContains(substr string) bool {
 	return strings.Contains(ReadKernelCmdline(), substr)
 }
+
+func CheckIfFileExistAndSize(input string, size int64) (bool, error) {
+	fileInfo, err := os.Stat(input)
+	if os.IsNotExist(err) {
+		return false, err
+	} else if fileInfo.Size() != size {
+		return false, errors.New("Size not matched")
+	} else {
+		return true, err
```

Comment:
We shouldn't expect the err is nil here, the err could be another type of error other than file not exist. We should still check if the err is nil or not.